test/alternator/test_streams.py

```diff
@@ -636,13 +637,15 @@ def compare_events(expected_events, output, mode):
     # then we remove this matched even from expected_events_map)
     for event in output:
         # In DynamoDB, eventSource is 'aws:dynamodb'. We decided to set it to
-        # a *different* value - 'scylladb:alternator'. Issue #6931.
+        # a *different* value - 'scylladb:alternator'. Reproduces #6931.
```

Comment:
I think this particular comment should still be issue 6931 or see 6931 - **not** reproduces 6931. In this context, eventSource, we do not reproduce 6931 - quite the opposite - we decided not to check eventSource so as not to reproduce this issue.

The phrase "Reproduces 6931" is fine in your new comments below, where we do check the region.

Reply:
Done.